src/pages/CreatorCenter/components/Main/components/MainContent/components/ClassListSearchBar/ClassListSearchBar.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import styled from 'styled-components';
+
+function ClassListSearchBar(props) {
+  return (
+    <SearchBar>
+      <SearchInput />
+      <SearchIcon />
+    </SearchBar>
+  );
+}
+
+const SearchBar = styled.div`
+  ${props => props.theme.variables.flex('', 'space-between', 'center')}
+  width: 100%;
+  padding: 0 10px;
+  height: 40px;
+  background-color: #f2f2f2;
+  border-radius: 5px;
+`;
+
+const SearchInput = styled.input.attrs({
+  placeholder: '상품명 검색',
+})`
+  padding-left: 5px;
+  background-color: transparent;
+  outline: none;
+  border: none;
+  height: 100%;
+  width: 100%;
+`;
```

Comment:
```suggestion
const SearchInput = styled.input`
  padding-left: 5px;
  background-color: transparent;
  outline: none;
  border: none;
  height: 100%;
  width: 100%;
`;
```

위를 수정하면 이렇게 작성할 수 있겠네요!